### PR TITLE
Make release script write correct no-op changelog

### PR DESCRIPTION
**Pull request by erikjohnston**
As we want to include the previous version in the "No new changes..." string.

### CODE REVIEW

File: scripts-dev/release.py [hunk 1]
```diff
@@ -523,5 +525,29 @@ class VersionSection:
...
+    with open("CHANGES.md", "r+") as f:
+        existing_content = f.read()
+        f.seek(0, 0)
+        f.write(new_changes)
+        f.write("\n")
+        f.write(existing_content)
+
+    # Remove all the news fragments
+    for f in glob.iglob("changelog.d/*.*"):
+        os.remove(f)
```
Comment (anoadragon453) [inline scripts-dev/release.py:549]: Would this delete https://github.com/matrix-org/synapse/blob/develop/changelog.d/.gitignore?

**Review by anoadragon453 — COMMENTED** [review]
lgtm besides one small issue.

File: scripts-dev/release.py (hunk 1 above)
Comment (erikjohnston) [reply scripts-dev/release.py:549]: Nope, glob ignores `.*` files by default

**Review by anoadragon453 — APPROVED** [review]
lgtm then!

**Pull request merged**